include/st_convert_api.h

```diff
@@ -998,6 +998,20 @@ static inline int st20_rfc4175_444le12_to_gbrp12le(struct st20_rfc4175_444_12_pg
   return st20_rfc4175_444le12_to_444p12le(pg, g, r, b, w, h);
 }
 
+// Convert rfc4175_422be10 to yuv422p16le with 6 bit padding (no optimization so far).
```

Comment:
Comment `no optimization so far` is invalid as you wrote AVX512 optimized code :)
Could you remove this comment is all 4 occurrences  (2 x in st_convert_api.h and 2x in st_convert_internal.h)